cli/azd/pkg/infra/provisioning/bicep/bicep_provider.go

```diff
@@ -213,35 +213,35 @@ func (p *BicepProvider) State(ctx context.Context) (*StateResult, error) {
 var ResourceGroupDeploymentFeature = alpha.MustFeatureKey("resourceGroupDeployments")
 
 // Plans the infrastructure provisioning
-func (p *BicepProvider) Plan(ctx context.Context) (*DeploymentPlan, error) {
+func (p *BicepProvider) plan(ctx context.Context) (*Deployment, *BicepDeploymentDetails, error) {
```

Comment:
Just wondering - with this change can we stop exporting `BicepDeploymentDetails` from this package or do we still need it to be exported because we use it outside this package?

Reply:
Yes, 
Updated.
Also updated the TerraformDetails